src/SerEvaluator.cs

```diff
@@ -630,8 +614,8 @@ private void FindTemplatePaths(UserParameter parameter, SerConfig config, string
                 }
                 else if (templateUri.Scheme.ToLowerInvariant() == "lib")
                 {
-                    var connections = GetConnections(onDemandConfig.Connection.ServerUri, parameter.AppId, cookie);
-                    dynamic libResult = connections?.FirstOrDefault(n => n["qName"]?.ToString()?.ToLowerInvariant() == result.Item2) ?? null;
+                    var connections = parameter.SocketConnection.GetConnectionsAsync().Result;
```

Comment:
don't use this Sync Calls for .Result -> Task -> ContuineWith or Async / Await

Reply:
Is Implemented